shopinvader_schema_sale/__manifest__.py

```diff
@@ -13,6 +13,7 @@
     "depends": [
         "sale_cart",
         "sale_discount_display_amount",
+        "shopinvader_sale_state",
```

Comment:
@sebastienbeau I would prefer to have a 'shopinvader_schema_sale_state' with a depends on 'shopinvader_schema_sale' and 'shopinvader_sale_state'. In this way we keep the 'original' state by default and we've an addon that replace this behaviour with a more specific one.

Reply:
@lmignon done

Reply:
@lmignon done